build.txt

```diff
@@ -0,0 +1,48 @@
+/*
+Insert any copyrights you may have here.
+*/
+
+//https://github.com/OSSILE/RelicPackageManager
+
+name:
+    iOpen
+    
+monmsg:
+  //Ignore failed ADDLIBLE
+  CPF2103
+
+build:
+  //Delete existing source
+  CLRLIB &INTOLIB
+
+  ADDLIBLE LIB(&INTOLIB)
+  
+  //Let's copy all the files
+  CRTSRCPF FILE(&INTOLIB/QCSRC) RCDLEN(112)
+  CPYFRMSTMF FROMSTMF('&DIR/crtstmf.c') TOMBR('/QSYS.lib/&INTOLIB.lib/QCSRC.file/CRTSTMF.mbr') MBROPT(*ADD)
```

Comment:
Not sure it really matters actually, seeing as I'm doing a CLRLIB on the library.

Thinking about it, I should not be doing a CLRLIB, I should be just replacing all the members and objects. I will change this ASAP.